### ISSUE TITLE
Allow running as 'heroku pg:pgcli' on current app

### ISSUE THREAD

**Issue by mariusa** [techgaun/heroku-pgcli#1]
Please allow this:
```
Open a pgcli shell to DATABASE_URL on the current app:

$ heroku pg:pgcli
```

similar to https://github.com/robanderton/heroku-pg-pgcli

**Comment by techgaun** [comment]
@mariusa thanks for opening the issue. I will look into this as I thought this was already the case.

**Comment by techgaun** [comment]
@mariusa I checked the source code again and we are defaulting to `DATABASE_URL`. `heroku pgcli -a <some_app>` should connect to `DATABASE_URL` by default.

**Comment by mariusa** [comment]
Right, that does work. What would be great is to be able to run without ```-a``` from current app folder, eg

```
heroku pgcli
```
similar to 
```
heroku pg:psql
```
or the other plugin mentioned in 1st comment.

**Comment by techgaun** [comment]
@mariusa ah got it, thats a fair ask. I will work on it sometimes soon. thanks for clarifying the issue.
